pylint/test/extensions/test_check_ungrouped_isinstances.py

```diff
@@ -0,0 +1,58 @@
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+# For details: https://github.com/PyCQA/pylint/blob/master/COPYING
+
+"""Tests for the pylint checker in :mod:`pylint.extensions.check_ungrouped_isinstances
+"""
+
+import os.path as osp
+import unittest
+
+from pylint import checkers
+from pylint.extensions.check_ungrouped_isinstances import register
```

Comment:
I am considering moving this under the refactoring checker, so this test would be gone and we could use proper functional tests.